iso15118/evcc/controller/simulator.py

```diff
@@ -168,11 +173,14 @@ async def get_energy_transfer_mode(
         """Overrides EVControllerInterface.get_energy_transfer_mode()."""
         if protocol == Protocol.DIN_SPEC_70121:
             return EnergyTransferModeEnum.DC_EXTENDED
-        return EnergyTransferModeEnum.AC_THREE_PHASE_CORE
+        else:
+            return self.config.energy_transfer_mode
```

Comment:
Do we need the if check now? Couldn't we return self.config.energy_transfer_mode directly?

Reply:
as it is an enumeration of int, better to define a string to enum conversion.

Reply:
Done